src/components/course/CourseSidebar.jsx

```diff
@@ -64,7 +64,7 @@ const CourseSidebar = () => {
             content={partners.map(partner => (
               <span key={partner.key} className="d-block">
                 <Hyperlink
-                  destination={partner.marketingUrl}
+                  destination={`/${enterpriseConfig.slug}/search?partners.name=${partner.name}`}
```

Comment:
[question/sanity check] If we're no longer linking to external marketing website, should these hyperlinks continue to open in a new tab or should they rely on React routing at this point, e.g. using `Link` component from `react-router-dom`? If these links were to open in the same window, `Hyperlink` would trigger a full page refresh where all the JS/CSS/etc. assets need to be re-downloaded whereas `Link` would avoid a full page refresh.

Generally, links that open in the same domain (enterprise.edx.org) would open in the same tab versus opening in a new tab.

Reply:
not a bad idea, I was debating whether it be a new tab or not. I like your reasoning

Reply:
hah actually we do almost exactly this in some pathway code already 🙃

Reply:
lmk what you think!